security-analysis/security-analysis-api/src/main/java/com/powsybl/security/json/action/LoadActionDeserializer.java

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.security.json.action;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.deser.std.StdDeserializer;
+import com.powsybl.commons.json.JsonUtil;
+import com.powsybl.security.action.LoadAction;
+import com.powsybl.security.action.LoadActionBuilder;
+
+import java.io.IOException;
+
+/**
+ * @author Anne Tilloy <anne.tilloy@rte-france.com>
+ */
+public class LoadActionDeserializer extends StdDeserializer<LoadAction> {
+
+    public LoadActionDeserializer() {
+        super(LoadAction.class);
+    }
+
+    private static class ParsingContext {
+        String id;
+        String loadId;
+        boolean relativeValue;
```

Comment:
need to capture if relativeValue is not in input json (error will be raised when calling LoadActionBuilder#build())

```suggestion
        Boolean relativeValue;
```